python/turbodbc_test/test_executemanycolumns.py

```diff
@@ -131,71 +169,102 @@ def test_datetime64_nanoseconds_column(dsn, configuration):
+@for_each_column_backend
 @for_each_database
-def test_string_column(dsn, configuration):
+def test_string_column(dsn, configuration, column_backend):
     _full_column_tests(configuration,
                        "INSERT STRING",
                        ["Simple", "Non-unicode", "Strings"],
-                       'object')
+                       'object',
+                       column_backend)
 
 
+@for_each_column_backend
 @for_each_database
-def test_unicode_column(dsn, configuration):
+def test_unicode_column(dsn, configuration, column_backend):
     _full_column_tests(configuration,
                        "INSERT UNICODE",
                        [u"a\u2665\u2665\u2665\u2665\u2665", u"b\u2665", u"c\u2665\u2665\u2665"],
-                       'object')
+                       'object',
+                       column_backend)
 
 
-def _test_none_in_string_column(configuration, fixture):
+def _test_none_in_string_column(configuration, fixture, column_backend):
     with open_cursor(configuration) as cursor:
         with query_fixture(cursor, configuration, fixture) as table_name:
-            columns = [array([None], dtype='object')]
+            columns = _to_columns([None], 'object', column_backend)
             cursor.executemanycolumns("INSERT INTO {} VALUES (?)".format(table_name), columns)
 
             results = cursor.execute("SELECT A FROM {} ORDER BY A".format(table_name)).fetchall()
             assert results == [[None]]
 
 
+@for_each_column_backend
 @for_each_database
-def test_string_column_with_None(dsn, configuration):
-    _test_none_in_string_column(configuration, "INSERT STRING")
+def test_string_column_with_None(dsn, configuration, column_backend):
+    _test_none_in_string_column(configuration, "INSERT STRING", column_backend)
 
 
+@for_each_column_backend
 @for_each_database
-def test_unicode_column_with_None(dsn, configuration):
-    _test_none_in_string_column(configuration, "INSERT UNICODE")
+def test_unicode_column_with_None(dsn, configuration, column_backend):
+    _test_none_in_string_column(configuration, "INSERT UNICODE", column_backend)
 
 
+@for_each_column_backend
 @for_each_database
-def test_multiple_columns(dsn, configuration):
+def test_multiple_columns(dsn, configuration, column_backend):
     with open_cursor(configuration) as cursor:
         with query_fixture(cursor, configuration, 'INSERT TWO INTEGER COLUMNS') as table_name:
             columns = [array([17, 23, 42], dtype='int64'), array([3, 2, 1], dtype='int64')]
+            if column_backend == 'arrow':
+                columns = [pa.Array.from_pandas(x) for x in columns]
+                columns = pa.Table.from_arrays(columns, ['column1', 'column2'])
             cursor.executemanycolumns("INSERT INTO {} VALUES (?, ?)".format(table_name), columns)
 
             results = cursor.execute("SELECT A, B FROM {} ORDER BY A".format(table_name)).fetchall()
             assert results == [[17, 3], [23, 2], [42, 1]]
 
 
+@for_each_column_backend
```

Comment:
I have moved corner cases such as this to a separate test file: https://github.com/blue-yonder/turbodbc/blob/master/python/turbodbc_test/test_executemanycolumns_corner_cases.py#L64